### PR TITLE
chore: fix ruff ARG002 and ANN401

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

**Review by wuwentao — APPROVED** [review]

File: custom_components/midea_ac_lan/config_flow.py [hunk 1]
```diff
@@ -270,11 +269,13 @@ async def async_step_discovery(
...
+                    str(device_id),
+                    device[CONF_IP_ADDRESS],
                 ):
-                    self.available_device[device_id] = (
-                        f"{device_id} ({self.supports.get(device.get(CONF_TYPE))})"
-                    )
+                    self.available_device[
+                        device_id
+                    ] = f"{device_id} ({self.supports.get(
+                        device.get(CONF_TYPE))})"
```
Comment (Necroneco) [inline custom_components/midea_ac_lan/config_flow.py:278]: Python version 3.11 does not allow new lines in expression parts of non-triple-quoted f-strings

Comment (chemelli74) [reply custom_components/midea_ac_lan/config_flow.py:278]: Resolved in upcoming PRs

Comment (Necroneco) [reply custom_components/midea_ac_lan/config_flow.py:278]: Sad, this hasn't been resolved

Comment (chemelli74) [reply custom_components/midea_ac_lan/config_flow.py:278]: It was: <https://github.com/wuwentao/midea_ac_lan/commit/63e801d05d08a1f41f6213e2abf7430169ce7315>

**Pull request merged**